pyo3-macros-backend/src/pyfunction.rs

```diff
@@ -183,9 +188,50 @@ pub fn parse_name_attribute(attrs: &mut Vec<syn::Attribute>) -> syn::Result<Opti
 pub fn build_py_function(ast: &mut syn::ItemFn, args: PyFunctionAttr) -> syn::Result<TokenStream> {
     let python_name =
         parse_name_attribute(&mut ast.attrs)?.unwrap_or_else(|| ast.sig.ident.unraw());
+
     add_fn_to_module(ast, python_name, args)
 }
 
+/// Parses #[pyo3(from_python_with = "func")]
+pub fn parse_arg_attributes(cap: &mut syn::PatType) -> syn::Result<PyFunctionArgAttr> {
```

Comment:
How about taking `&mut Vec<syn::Attribute>` instead of PatType?
Then it's obvious what is mutated.

Reply:
Thanks for the tip! Fixed